governance/reviews/template.md

```diff
@@ -38,16 +60,20 @@ The following aspects of governance are exemplary, and can be referenced as exam
 ### Areas for Improvement
 
 Over the next year, the project should work on the following issues to improve its governance, these are considered non-blocking:
-<!--- This is a summarized listing of longer term improvement areas for the project. These items are strongly encouraged but not required for the project's maturity level. Fully detailed descriptions are found in the Finding Table. Items listed here should be in priority ordering and would be considered Medium and Low. --->
+<!--- This is a summarized listing of longer term improvement areas for the project. These items are strongly encouraged but not required for the project's maturity level. Fully detailed descriptions are found in the Finding Table. Items listed here should be in priority ordering. Items in the "Governance Findings Table" at the bottom except the "Critical" importance should be reported here. -->
 
 *
 
+Details of these issues can be found in the [Findings Table](#Governance-Findings-Table) and the related sections below.
+
 ## Review
 
 ### Governance Description
 
 <!--- Narrative describing the governance type of the project, some general information about its leadership, and the project's general status and maturity. If the project has any unusual aspects to its governance, describe them here.  Link to the project's existing documents where applicable. --->
 
+<!--- You will most likely need the number of substantial contributors in the project and write it down along with their employer affiliations. If project is already under CNCF, use Devstats; if not, manually check the repositories. --->
```

Comment:
I'm not sure about this. There isn't a requirement to document substantial contributors, but there is a requirement for having maintainers from at least 2 companies, so I focus on the maintainers. I also usually start by getting maintainer affiliations from the governance docs, since most of them are listed there. I think it's harder to pull this out of devstats for individual contributors. Also, I don't think we've done any governance assessments where the project isn't already under the CNCF. I guess maybe it could happen if we're asked to assess a project that is being proposed to come in as an incubating project?

Reply:
> Also, I don't think we've done any governance assessments where the project isn't already under the CNCF.
I've done it for Flatcar: https://github.com/cncf/tag-contributor-strategy/blob/main/governance/assessments/projects/flatcar/2023-08-15.md

> I guess maybe it could happen if we're asked to assess a project that is being proposed to come in as an incubating project?

I've only done a single review so far. After my single review, I thought this is a general thing to do and wanted to document what I've done for any future reviewers. Thanks catching this!

I will update this instruction to make it only applies for projects that want to apply to CNCF.

Reply:
Removed!